modules/learn/pages/security/using-multiple-cas.adoc

```diff
@@ -0,0 +1,72 @@
+= Using Multiple Root Certificates
+:description: Couchbase Server supports use of multiple CA (or 'root') certificates, for a single cluster.
+
+[abstract]
+{description}
+
+== Benefits
+
+Prior to Couchbase Server Version 7.1, a Couchbase-Server cluster could maintain at most _one_ root certificate: all nodes and all clients were obliged to rely on this certificate's authority.
+From version 7.1, Couchbase Server permits multiple root certificates to maintained in a _trust store_ for the cluster.
+This:
```

Comment:
Please make sure the following use-cases are covered here.
Couchbase-server uses "trusted certificate list" to:

- verify client certificates (when client certificates authentication is enabled)
- verify identities of cluster nodes when n2n encryption is enabled
- verify identities of nodes that join cluster(when cluster is provisioned with certificates; when self-generated certificates are used we don't verify new-node's identity);
- verify identity of LDAP servers (when TLS is turned on in LDAP settings)

please let me know if you have questions.

Reply:
I have added these cases.